docs/tr2/CHANGELOG.md

```diff
@@ -33,7 +33,7 @@
 - fixed sprite pickups not being paused in the pause/inventory screen (#2319, regression from 0.6)
 - fixed Skidoo snow wake effects at slow speeds (#2324, regression from 0.6)
 - fixed software renderer skybox occlusion issues (#2343, regression from 0.7)
-- fixed Floating Islands losing some frames if played first (#2325, regression from 0.7)
+- improved slightly rendering performance in big rooms (#2325)
```

Comment:
Idk if you wan to match TR1X's changelog or not, but this wording is a little awkward. Normally it's `slightly improved` but you want the verb first in the changelog. I think `improved rendering performance slightly in big rooms` would work. Sorry for the nitpick. Great fix!

Reply:
Oh, thank you for spotting this! Indeed, I had a better wording for this, but it got lost in the rebases. I'll try to recover it, since I need to do a small fix anyway.

Reply:
Fixed